py/client-ticking/setup.py

```diff
@@ -59,9 +59,11 @@ def _compute_version():
     ext_modules = cythonize(
```

Comment:
Commenting for something I see above (not in review). Should we add 3.12 classifier?

Reply:
Thanks